packages/oscal-react-library/src/components/OSCALBackMatter.tsx

```diff
@@ -215,6 +215,10 @@ function BackMatterResource(props: BackMatterResourceProps) {
           action={<CitationDisplay resource={resource} />}
         />
         <CardContent>
+          <OSCALPropertiesDialog
+            properties={resource?.props}
+            title={resource?.title ?? resource?.uuid}
```

Comment:
Let's not use a UUID in the title... I'd rather see us do something like `<NotSpecifiedTypography component="span">Resource</NotSpecifiedTypography>`

Reply:
I added a fallback title within `Properties`, maybe we would want to make it more generic within Properties. Maybe it could just be "Not Specified Properties"?

Reply:
Sorry, I should have mentioned I corrected this to what you were suggesting (`<NotSpecifiedTypography component="span">Resource</NotSpecifiedTypography>`)